src/main/java/frc/robot/RobotContainer.java

```diff
@@ -74,8 +75,13 @@ public RobotContainer(RobotId robotId) {
       case TestBed2:
         intake = new Intake();
         index = new StorageIndex();
-        //SmartDashboard.putData("IntakeNote", new IntakeNote(intake, index, arm));
         break;
+      case Vertigo:
```

Comment:
Make Vertigo the same as SwerveTest by adding case Vertigo: above or below case SwerveTest: without a break;